src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/jira/flow/JiraNodeFlow.kt

```diff
@@ -10,15 +10,14 @@ import java.util.*
 import java.util.concurrent.ConcurrentLinkedQueue
 
 @ThreadSafe
-class JiraNodeFlow {
-
+class JiraNodeFlow : TcpServerFlow {
     private val preInstallHooks: Queue<TcpServerHook> = ConcurrentLinkedQueue()
     private val postInstallHooks: Queue<InstalledJiraHook> = ConcurrentLinkedQueue()
-    private val preStartHooks: Queue<InstalledJiraHook> = ConcurrentLinkedQueue()
+    private val preStartHooks: Queue<PreStartHook> = ConcurrentLinkedQueue()
```

Comment:
`PreStartHook` is inconsistently named when compared to `TcpServerHook`, etc.
The old hook types describe their dependency - `TcpServerHook` depends on `TcpServer`, `InstalledJiraHook` depends on `InstalledJira`.
If hooks are supposed to describe a point in time, they should be called so. E.g. `InstalledJiraHook` should be renamed to `PostInstallHook`.